src/Microsoft.OData.Core/ODataInnerError.cs

```diff
@@ -30,17 +32,44 @@ public ODataInnerError()
         /// <summary>Initializes a new instance of the <see cref="T:Microsoft.OData.ODataInnerError" /> class with exception object.</summary>
         /// <param name="exception">The <see cref="T:System.Exception" /> used to create the inner error.</param>
         public ODataInnerError(Exception exception)
+            : this(exception, "internalexception")

```

Comment:
>"internalexception" [](start = 30, length = 19)

This should be a defined constant, like JsonConstants.ODataErrorInnerErrorInnerErrorName (except not JSON specific).